protoc-gen-swagger/genswagger/template.go

```diff
@@ -169,10 +180,9 @@ func nestedQueryParams(message *descriptor.Message, field *descriptor.Field, pre
 				break
 			}
 		}
-
 		param := swaggerParameterObject{
 			Description: desc,
-			In:          "query",
+			In:          strings.ToLower(paramOpt.In.String()),
```

Comment:
doesn't this have to be `query` still? its inside `nestedQueryParams` and you verify above that its not `path` (though i guess this could still be `header` or `cookie` since you don't check those)

Reply:
Yeah the method name is confusing (since protoc-gen-swagger hardcodes "query" as `in`), but if I understand correctly having a dynamic `in` field should still work with this code. I can rename the method to `nestedNonPathOrBodyParams` (or something like that)? Maybe I'm missing something though and behavior needs to change based on the value of the `in` field.